test/Sema/package_resilience_super_class.swift

```diff
@@ -0,0 +1,63 @@
+// RUN: %empty-directory(%t)
+// RUN: split-file %s %t
+
+/// Access a package class (non-resilient type) subclassing a resilient class from a resilient module.
+// RUN: %target-build-swift-dylib(%t/%target-library-name(Utils)) %t/Utils.swift -module-name Utils -package-name mypkg -emit-module -emit-module-path %t/Utils.swiftmodule -enable-library-evolution
+// RUN: %target-build-swift -I %t -L %t -lUtils %s -o %t/main %target-rpath(%t) -package-name mypkg
+// RUN: %target-run %t/main %t/%target-library-name(Utils) | %FileCheck %s
```

Comment:
Hey @elsh, this test should be marked as `REQUIRES: executable_test` since it runs target executables.

Reply:
Thx https://github.com/apple/swift/pull/69863